src/com/sap/piper/MtaUtils.groovy

```diff
@@ -21,14 +21,15 @@ class MtaUtils {
 
         def dataFromJson = script.readJSON file: srcPackageJson
 
-        def mtaData  = script.readYaml text: script.libraryResource('template_mta.yml')
+        def mtaData  = script.readYaml text: script.libraryResource('template_mta.yaml')
```

Comment:
I believe this is false. There is a resource [template.yml](https://github.com/SAP/jenkins-library/blob/master/resources/template_mta.yml), but there is no corresponding resource for appendix `.yaml`.

I would like to suggest to keep the template as it is, other option would be to rename the resource accordingly.